src/main/scala/com.snowplowanalytics.sauna/SaunaOptions.scala

```diff
@@ -30,34 +30,40 @@ import com.sksamuel.avro4s._
 import play.api.libs.json._
 
 // sauna
-import responders._
 import loggers._
 import observers._
+import responders._
 
 /**
  * Options parsed from command line
- * 
+ *
  * @param configurationLocation root to directory with all configuration files
  */
 case class SaunaOptions(configurationLocation: File) {
+
   import SaunaOptions._
 
   /**
    * Parse and extract global configuration object from `configurationLocation`
    */
   def extract: SaunaSettings =
     SaunaSettings(
-      getConfig[AmazonDynamodbConfig],
-      getConfig[HipchatConfig],
-      getConfig[OptimizelyConfig],
-      getConfig[SendgridConfig],
-      getConfigs[LocalFilesystemConfig],
-      getConfigs[AmazonS3Config])
+      getConfig[AmazonDynamodbConfig_1_0_0],
```

Comment:
Cam we all `.loggers`/`.responders` to all classes, so it could look consistent?